public/app/features/plugins/plugin_loader.ts

```diff
@@ -218,3 +228,21 @@ export async function importAppPlugin(meta: PluginMeta): Promise<AppPlugin> {
 
   return plugin;
 }
+
+// exported for testing only
+export async function addLocalesToI18n(pluginId: string, locales: Record<string, string>): Promise<void[]> {
```

Comment:
likewise, I think we should avoid calling this 'locale'

Reply:
yup, will change this too 👍

Reply:
Fixed 👍